keyboards/peranekofactory/tone/rev2/config.h

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2021 peraneko
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x0000
+#define DEVICE_VER   0x0001
+#define MANUFACTURER PeranekoFactory
+#define PRODUCT      tone
+
+/* key matrix size */
+#define MATRIX_ROWS 1
+#define MATRIX_COLS 10
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ * 
+ */
+
+#define MATRIX_ROW_PINS { NO_PIN }
+#define MATRIX_COL_PINS { D4,C6,D7,E6,F6,F7,B1,B3 }
```

Comment:
Also should be using direct pins.

Reply:
Changed to use Direct_Pins in the relevant part.
It has been confirmed to work on the actual machine.